examples/billing/get_billing_setup.py

```diff
@@ -37,47 +35,41 @@ def main(client, customer_id, page_size):
         'billing_setup.payments_account_info.secondary_payments_profile_id '
         'FROM billing_setup')
 
-    results = ga_service.search(customer_id, query=query, page_size=page_size)
+    response = ga_service.search_stream(customer_id, query=query)
 
     try:
         # Use the enum type to determine the enum name from the value.
-        billing_setup_status_enum = (
-            client.get_type('BillingSetupStatusEnum', version='v3')
-                .BillingSetupStatus)
+        billing_setup_status_enum = client.get_type(
+            'BillingSetupStatusEnum', version='v3').BillingSetupStatus
 
         print('Found the following billing setup results:')
-        for row in results:
-            billing_setup = row.billing_setup
-            payments_account_info = billing_setup.payments_account_info
-            print('Billing setup with ID "%s", status "%s", '
-                  'payments_account "%s", payments_account_id "%s", '
-                  'payments_account_name "%s", payments_profile_id "%s", '
-                  'payments_profile_name "%s", '
-                  'secondary_payments_profile_id "%s".'
-                  % (billing_setup.id.value,
-                     billing_setup_status_enum.Name(billing_setup.status),
-                     billing_setup.payments_account.value,
-                     payments_account_info.payments_account_id.value,
-                     payments_account_info.payments_account_name.value,
-                     payments_account_info.payments_profile_id.value,
-                     payments_account_info.payments_profile_name.value,
-                     payments_account_info.secondary_payments_profile_id.value))
-    except google.ads.google_ads.errors.GoogleAdsException as ex:
-        print('Request with ID "%s" failed with status "%s" and includes the '
-              'following errors:' % (ex.request_id, ex.error.code().name))
+        for batch in response:
+            for row in batch.results:
+                billing_setup = row.billing_setup
+                pai = billing_setup.payments_account_info
+                print(f'Billing setup with ID {billing_setup.id.value}, '
+                      f'status "{billing_setup_status_enum.Name(billing_setup.status)}", '
+                      f'payments_account "{billing_setup.payments_account.value}" '
+                      f'payments_account_id "{pai.payments_account_id.value}", '
```

Comment:
ID doesn't need quotes. Same for secondary_payments_profile_id below.

Reply:
Note that if `secondary_payments_profile_id` is empty (as it is in my case), the result is just a blank space.

Reply:
That's why we have [this logic](https://github.com/googleads/google-ads-php/blob/master/examples/Billing/GetBillingSetup.php#L141) in PHP's. :)